mon-pix/app/routes/campaigns/skill-review.js

```diff
@@ -13,4 +13,9 @@ export default Route.extend(AuthenticatedRouteMixin, {
       assessment: store.findRecord('assessment', assessmentId)
     });
   },
+
+  async afterModel(model) {
+    const canFindANextChallenge = await this.store.queryRecord('challenge', { assessmentId: model.assessment.id, tryImproving: true });
+    this.controllerFor('campaigns.skill-review').set('displayImprovementButton', canFindANextChallenge ? true : false);
```

Comment:
`this.controllerFor('campaigns.skill-review').set('displayImprovementButton', canFindANextChallenge)` ?

Reply:
Dans le cas présent, on manipule un objet et non un `boolean` donc je simplifie par 
```js
this.controllerFor('campaigns.skill-review').set('displayImprovementButton', !!canFindANextChallenge)
```
https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript

J'en profite pour éclaircir l'intention, récupérer un "next `Challenge` que l'on peut améliorer", en la renommant la variable par `improvableNextChallenge`